gradle/docker.gradle

```diff
@@ -29,6 +44,7 @@ ext {
     dockerRetries = project.hasProperty('dockerRetries') ? dockerRetries.toInteger() : 3
     dockerBinary = project.hasProperty('dockerBinary') ? [dockerBinary] : ['docker']
     dockerBuildArg = ['build']
+    dockerArch = project.hasProperty('dockerArch') ? dockerArch : detectDockerArch()
```

Comment:
I don't get why we need a detect, the build machine might not be the machine running the containers.
I would just kip this detection, and just use dockerArch when it's define and passed in.

Reply:
since there is only one arch for now, I would just make is explicit the parameter s390x as a flag/property

Reply:
Hmm... I see your point, but it opens a big can of worms about where the build is running vs where the target machine is.  This worked great for logging onto the LoZ environment and typing './gradlew distDocker' before running the ansible local configuration.  The detect is useful because it ensure the architecture of the Dockerfile matches to where the build will be run.  The gradle script doesn't really have any insight into target architecture, and passing it a dockerArch that mis-matches the Docker it's running on will have core-generating results.

I can take it or leave it -- removing it just adds one more parameter to remember to get right when running './gradlew'.

Reply:
Also, 'ppc64le' is running close on the heels of 's390x'.  We could create a flag for each architecture, I suppose.